.ghcid

```diff
@@ -1 +0,0 @@
--c "stack ghci --test --ghci-options=-fno-code --ghci-options=-fno-break-on-exception --ghci-options=-fno-break-on-error --ghci-options=-v1 --ghci-options=-ferror-spans --ghci-options=-j" --restart=stack.yaml --restart=git-remote-ipfs/git-remote-ipfs.cabal --restart=ipfs-api/ipfs-api.cabal --restart=ipld-cid/ipld-cid.cabal --restart=multibase/multibase.cabal --restart=multihash-cryptonite/multihash-cryptonite.cabal --restart=multihash-serialise/multihash-serialise.cabal
```

Comment:
If you want, it's still possible to use `ghcid` with `cabal-new`, just replace `stack ghci` with `cabal new-repl` 😀 

YMMV though, as you would probably have to pick a target as `cabal new-*` works slightly differently, so we could just as well get rid of this one, for now.

Reply:
Practically, it's unusable for a monorepo - if you pick a target, it will not notice changes in local source dependencies.

I'm still using stack with ghcid here, but removed this file to convey "not officially maintained" ;)